app.js

```diff
@@ -25,18 +28,21 @@ const sessionSettings = {
 // register all partials from directory.
 hbs.registerPartials("./views/partials");
 
-app.set("views", "./views");
+app.set("views", path.join(__dirname, "views"));
```

Comment:
Actually, require is different. Within require, ./ actually refers to the current directory.